08-interop/test_spec08.py

```diff
@@ -0,0 +1,88 @@
+import re
+import sys
+
+import pexpect.replwrap
+import pytest
+
+import riotctrl.shell
+from riotctrl_shell.gnrc import GNRCICMPv6Echo
+from riotctrl_shell.netif import Ifconfig
+
+from testutils.native import bridge, get_link_local, get_ping_cmd, \
+                             interface_exists
+from testutils.shell import lladdr, ping6, GNRCUDP
+
+
+GNRC_APP = 'examples/gnrc_networking'
+LWIP_APP = 'tests/lwip'
+pytestmark = pytest.mark.rc_only()
+
+
+class Shell(Ifconfig, GNRCICMPv6Echo, GNRCUDP):
+    pass
+
+
+@pytest.mark.skipif(not interface_exists("tap0"),
+                    reason="tap0 does not exist")
+# nodes passed to riot_ctrl fixture
+@pytest.mark.parametrize('nodes',
+                         [pytest.param(['native'])],
+                         indirect=['nodes'])
+def test_task01(riot_ctrl, log_nodes):
+    node = riot_ctrl(0, GNRC_APP, Shell, port='tap0')
+    linux = pexpect.replwrap.bash()
+
+    node_iface, node_addr = lladdr(node.ifconfig_list())
+    assert node_addr.startswith("fe80::")
+    linux_iface = bridge('tap0')
+    linux_addr = get_link_local(linux_iface)
```

Comment:
Maybe when the other tasks get implemented this will make sense in a common module

Reply:
I don't understand? Those tasks would also be implemented also in this script, so no need to move that function to a separate `common` module.

Reply:
ah true. nvm